lib/src/widgets/reactive_form.dart

```diff
@@ -67,10 +63,13 @@ class ReactiveForm extends StatelessWidget {
     return FormControlInheritedStreamer(
       control: formGroup,
       stream: formGroup.statusChanged,
-      child: WillPopScope(
-        onWillPop: onWillPop,
-        child: child,
-      ),
+      child: canPop != null || onPopInvoked != null
+          ? ReactiveFormPopScope(
+              canPop: canPop,
+              onPopInvoked: onPopInvoked,
+              child: child,
+            )
```

Comment:
Thanks for the contribution @remonke.

Why create a new widget instead of using directly the `PopScope` here?

Reply:
This is the way they do it in the Flutter `Form` widget, just decided to copy this.

Reply:
It would also improve performance I guess, since each form change would cause the widget to get rebuild.